src/app/shared/store/registration.state.ts

```diff
@@ -1,57 +1,56 @@
-import { ParentService } from '../services/parent/parent.service';
-import { Parent } from '../models/parent.model';
+import { Observable, of } from 'rxjs';
 import { catchError, tap } from 'rxjs/operators';
-import { Router } from '@angular/router';
+
+import { TerritorialCommunityAdmin } from 'shared/models/territorialCommunityAdmin.model';
+import { TerritorialCommunityAdminService } from 'shared/services/territorial-community-admin/territorial-community-admin.service';
+import { ModeConstants } from '../constants/constants';
+import { SnackbarText } from '../enum/enumUA/messageBer';
 import { Role } from '../enum/role';
-import { UserService } from '../services/user/user.service';
-import { Observable, of } from 'rxjs';
-import { TechAdminService } from '../services/tech-admin/tech-admin.service';
-import { MarkFormDirty, ShowMessageBar } from './app.actions';
-import { HttpErrorResponse } from '@angular/common/http';
-import { TechAdmin } from '../models/techAdmin.model';
+import { Parent } from '../models/parent.model';
 import { Provider } from '../models/provider.model';
+import { RegionAdmin } from '../models/regionAdmin.model';
+import { TechAdmin } from '../models/techAdmin.model';
+import { User } from '../models/user.model';
 import { MinistryAdminService } from '../services/ministry-admin/ministry-admin.service';
-import { ModeConstants } from '../constants/constants';
-import { Location } from '@angular/common';
-import { SnackbarText } from '../enum/enumUA/messageBer';
+import { ParentService } from '../services/parent/parent.service';
+import { ProviderService } from '../services/provider/provider.service';
 import { RegionAdminService } from '../services/region-admin/region-admin.service';
-import { RegionAdmin } from '../models/regionAdmin.model';
-import { TerritorialCommunityAdmin } from 'shared/models/territorialCommunityAdmin.model';
+import { TechAdminService } from '../services/tech-admin/tech-admin.service';
+import { UserService } from '../services/user/user.service';
+import { MinistryAdmin } from './../models/ministryAdmin.model';
+import { MarkFormDirty, ShowMessageBar } from './app.actions';
 import {
-  TerritorialCommunityAdminService
-} from 'shared/services/territorial-community-admin/territorial-community-admin.service';
+  CheckAuth,
+  CheckRegistration,
+  GetProfile,
+  GetUserPersonalInfo,
+  Login,
+  Logout,
+  OnAuthFail,
+  OnUpdateUserFail,
+  OnUpdateUserSuccess,
+  UpdateUser
+} from './registration.actions';
 
 export interface RegistrationStateModel {
   isAuthorized: boolean;
   isLoading: boolean;
-  isAutorizationLoading: boolean;
+  isAuthorizationLoading: boolean;
   user: User;
   provider: Provider;
   parent: Parent;
   techAdmin: TechAdmin;
-  regionAdmin: RegionAdmin;
   ministryAdmin: MinistryAdmin;
-  territorialCommunityAdmin: TerritorialCommunityAdmin;
+  regionAdmin: RegionAdmin;
+  areaAdmin: TerritorialCommunityAdmin;
```

Comment:
You add `areaAdmin` here, but do not replace `territorialCommunityAdmin` below (line 216)